README.md

```diff
@@ -40,14 +44,23 @@ You can also install packages from your distribution.
 
 ## Running the game
 
-Start the server on some machine:
+Run the game using Makefile with the command,
+
+	    make server
+  
+
+Alternatively, Start the server on some machine:
 
     ./rose-server
 
 Open a browser at http://\<server-address\>:8880 to view and control the game.
 
 Start up to 2 clients:
 
+	    make client
```

Comment:
Unfortunately this does not work - the client needs a driver module argument.

I think we need to remove the makefile instead of updating the readme. It helps only with the initial setup, and using a makefile breaks compatibility on windows.